homeassistant/components/deconz/light.py

```diff
@@ -66,10 +64,8 @@ def async_add_group(groups):
         entities = []
 
         for group in groups:
-            if group.lights:
-                new_group = DeconzGroup(group, gateway)
-                entity_handler.add_entity(new_group)
-                entities.append(new_group)
+            if group.lights and gateway.option_allow_deconz_groups:
```

Comment:
Shouldn't we check `gateway.option_allow_deconz_groups` before entering the for-loop ?

Reply:
That's true. I blindly went after how it was previously done...